mainProject/services/userService.ts

```diff
@@ -0,0 +1,71 @@
+import { IUserRequestBody } from "../interfaces/user";
+import { v4 } from "uuid";
+import UserModel from "../models/user";
+import { Op } from "sequelize";
+
+class UserService {
+  async getAutoSuggestUsers(login: string, limit: number) {
+    const result = await UserModel.findAll({
+      where: {
+        is_deleted: false,
+        login: {
+          [Op.like]: `${login}%`,
+        },
+      },
+      limit,
+    });
+
+    return result.sort((firstObj, secondObj) => {
```

Comment:
That could be done by DB

Reply:
it seems that I completely forgot about this possibility :)